frontend/types/modules.d.ts

```diff
@@ -33,13 +35,25 @@ export interface Image extends SearchableModule {
   providerString: string;
   description: string;
   bgImgPosition: string;
+  meta?: Meta;
 }
 
 export interface ModuleWithImages {
   primaryImage: Image;
   cachedImages: Image[];
 }
 
+export interface Text {
+  text?: string[]; // for quotes.quote
+  description?: string[]; // for sources.source, entities.person
+  elaboration?: string[]; // for occurances
+}
+export interface Meta {
+  score: number;
+  highlight: Text;
+  sort: number[];
+}
+
```

Comment:
nit: Just for consistency we should eventually have a newline between these interface definitions, but that doesn't have to be part of this PR.